src/Common.am

```diff
@@ -73,3 +73,15 @@ COMPAT_LIB = $(top_builddir)/compat/libcompatsquid.la $(LIBPROFILER)
 
 ## Some helpers are written in Perl and need the local shell defined properly
 subst_perlshell = sed -e 's,[@]PERL[@],$(PERL),g' <$(srcdir)/$@.pl.in >$@ || ($(RM) -f $@ ; exit 1)
+
+EXTENSIONS=.hdrtest
+.h.hdrtest:
+	filepath=`echo $< | sed 's/^[/\.]*//' `; \
+	srcfile=`basename $@`; \
+		sed 's%[@]HEADER[@]%${<}%' "$(top_srcdir)/test-suite/testHeader.cc.in" >"$$srcfile.cc" && \
+		$(CXXCOMPILE) -DIN_TESTHEADERS=1 -c -o "$$srcfile" "$$srcfile.cc" ; \
```

Comment:
Looks odd that we require `sed` to succeed (to proceed to CXXCOMPILE), but we do not require CXXCOMPILE to succeed (to proceed to `if test...`). Is there another `&&` missing here? Or should that `&&` be removed?

Reply:
CXXCOMPILE success is captured in the following "if $$? statement. Let me see if there is an alternate statement that can work better

Reply:
Done. We could argue we do not need the pretty-print result status as we are not really using TAP